src/index.html

```diff
@@ -13,6 +13,62 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    
+    <div class="stars__section">
+
+      <article class="stars stars--0">
+
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
```

Comment:
Consider adding an empty line between sibling blocks of HTML for better readability, as suggested by the HTML Formatting checklist.